distributed/utils.py

```diff
@@ -1107,6 +1107,14 @@ def nbytes(frame, _bytes_like=(bytes, bytearray)):
             return len(frame)
 
 
+def is_writeable(frame):
+    """ Check whether frame is writeable """
+    try:
+        return not memoryview(frame).readonly
+    except TypeError:
+        return None
```

Comment:
This is a small nitpick but should this be False instead of None ?

Reply:
No it should return `None`. `True` means it is writeable, `False` means it is readonly, and `None` means we can't tell so ignore. This impacts whether we need to copy or not later. So `None` effectively means don't try to perform a copy to make it writeable or readonly.

Reply:
Added to the docstring of this function. Please let me know if that makes sense 🙂